task/08-objects-tasks.js

```diff
@@ -23,7 +23,11 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+    this.width = width;
+    this.height = height;
+    this.__proto__.getArea = function () {
```

Comment:
The function will be defined each time when Rectangle creates, so actually we don't have difference to "this.getArea = " in your case. 
Please do without __proto__, it is for debug only